template/.drone.star

```diff
@@ -0,0 +1,654 @@
+def main(ctx):
+  before = [
```

Comment:
Use the .drone.star from ocis-proxy. This is not up to date 😄

Reply:
aye aye